SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Activate Rust in order to recognize the uv command
```

Comment:
How is this about "activating Rust"? I understand you are just adding the uv binary to PATH.

Reply:
This comment is deprecated. I was previosly activating Cargo in that section. Changing it now